tests/library/har.spec.ts

```diff
@@ -56,7 +56,7 @@ it('should have browser', async ({ browserName, browser, contextFactory, server
   await page.goto(server.EMPTY_PAGE);
   const log = await getLog();
 
-  expect(log.browser!.name.toLowerCase()).toBe(browserName);
+  expect(log.browser!.name.toLowerCase()).toBe(browserName.toLowerCase());
```

Comment:
```suggestion
  expect(log.browser!.name).toBe(browserName);
```
I don't see why we need to lower case there, it should be one of our predefined browser names.